### ISSUE TITLE
Crash on IE

### ISSUE THREAD

**Issue by MakoveevAS** [jonataswalker/ol-contextmenu#115]
When user open contextmenu several times (5) it crash on IE.
This happens when method Сlear tries to remove submenu elements

**Comment by joaoces19** [comment]
Hi, first of all congrats on the great plugin that you created and thanks for your work.

I'am having the same issue that is after several click's to open the context menu, the plugin seems to crash. This only happens in ie11.

[context-menu crash ie11.zip](https://github.com/jonataswalker/ol3-contextmenu/files/1266644/context-menu.crash.ie11.zip)

I was able to workaround the problem using JQuery but wanted to know if you have any tips on how to solve the problem without using JQuery. I will happily contribute to the project if i can find a better solution.

`Html.prototype.removeMenuEntry = function removeMenuEntry (index) {
  var element = utils.find('#' + index, this.container.firstChild);
  //element && this.container.firstChild.removeChild(element);
  element && $(element).remove();
  delete this.Base.Internal.items[index];
};`

I replaced the line "element && this.container.firstChild.removeChild(element);" with the jquery function 
"element && $(element).remove();"
